contracts/evm/library/utils/Types.sol

```diff
@@ -81,4 +85,13 @@ library Types {
         bytes data;
     }
 
+    function createPersistentMessageEnvelope(bytes memory data) internal pure returns (XCallEnvelope memory) {
+        return XCallEnvelope(
+            PERSISTENT_MESSAGE_TYPE,
+            data,
+            new string[](0),
+            new string[](0)
+        );
+    }
```

Comment:
This is what i referred to in creating helper functions, can you create one for each message, and also one that takes source and destination for each.
Name could even be createPersistentMessage only.
Otherwise remove for now and add for all later.

Reply:
Since this is needed for tests, so added now, and others, I am planning to add in dapp development.